website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -0,0 +1,41 @@
+---
+id: nevergrad_sweeper
+title: Nevergrad Sweeper plugin
+sidebar_label: Nevergrad Sweeper plugin
+---
+This plugin provides a mechanism for Hydra applications to use [Nevergrad](https://github.com/facebookresearch/nevergrad) algorithms for the optimization of the parameters of any experiment.
```

Comment:
See new standard header badges in [colorlog](https://hydra.cc/docs/next/plugins/colorlog) and [joblib](https://hydra.cc/docs/next/plugins/joblib_launcher).
The Ax plugin docs needs to be updated.

Add what makes sense (things about a pip package can wait until I create one).
I added plugin source and example links as standardized badges for all official plugins.